_includes/content/private-registry-image-path.md

```diff
@@ -0,0 +1,128 @@
+{% endif -%}
+sed -ie "s?quay.io.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" tigera-prometheus-operator.yaml
+sed -ie "/serviceAccountName: calico-prometheus-operator/a \      imagePullSecrets:\n\      - name: $PRIVATE_REGISTRY_PULL_SECRET"  tigera-prometheus-operator.yaml
+```
+{% comment %} The second 'sed' should be removed once operator launches Prometheus & Alertmanager {% endcomment %}
+
+Before applying `custom-resources.yaml`, modify registry references to use your custom registry:
+
+```bash
+{% if page.registry != "quay.io/" -%}
+sed -ie "s?{{ page.registry }}.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" custom-resources.yaml
+{% endif -%}
+sed -ie "s?quay.io.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" custom-resources.yaml
+```
+{% comment %} The second 'sed' should be removed once operator launches Prometheus & Alertmanager {% endcomment %}
+
+For <b>Openshift</b>, after downloading all manifests modify the following to use your custom registry:
+
+```bash
+{% if page.registry != "quay.io/" -%}
+sed -ie "s?{{ page.registry }}.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/02-tigera-operator.yaml
+{% endif -%}
+sed -ie "s?quay.io.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/02-tigera-operator.yaml
+
+{% if page.registry != "quay.io/" -%}
+sed -ie "s?{{ page.registry }}.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/04-deployment-prometheus-operator.yaml
+{% endif -%}
+sed -ie "s?quay.io.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/04-deployment-prometheus-operator.yaml
+sed -ie "s?containers:?imagePullSecrets:\n        - name: tigera-pull-secret \n      containers:?" manifests/04-deployment-prometheus-operator.yaml
+
+{% if page.registry != "quay.io/" -%}
+sed -ie "s?{{ page.registry }}.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/01-cr-prometheus.yaml
+{% endif -%}
+sed -ie "s?quay.io.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/01-cr-prometheus.yaml
+sed -ie "s?nodeSelector:?imagePullSecrets:\n    - name: tigera-pull-secret \n  nodeSelector:?" manifests/01-cr-prometheus.yaml
+
+{% if page.registry != "quay.io/" -%}
+sed -ie "s?{{ page.registry }}.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/01-cr-alertmanager.yaml
+{% endif -%}
+sed -ie "s?quay.io.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/01-cr-alertmanager.yaml
+sed -ie "s?nodeSelector:?imagePullSecrets:\n    - name: tigera-pull-secret \n  nodeSelector:?" manifests/01-cr-alertmanager.yaml
+
+```
+>**Note:** Add the image pull secret for your `registry` to the secret `tigera-pull-secret`
+{: .alert .alert-info }
+
+#### Configure the operator to use images from your private registry image path.
+
+Set the `spec.registry` and `spec.imagePath` field of your Installation resource to the name of your custom registry. For example:
+
+<pre>
+apiVersion: operator.tigera.io/v1
+kind: Installation
+metadata:
+  name: default
+spec:
+  variant: TigeraSecureEnterprise
+  imagePullSecrets:
+    - name: tigera-pull-secret
+  <b>registry: myregistry.com</b>
+  <b>imagePath: my-image-path</b>
```

Comment:
Is this line the only difference between the two docs? Feels like this could all just be a single document with an optional section for if you want to push your images to a different path. Or even a "Note" about it.

Reply:
The difference also includes different tagging, pushing, and seds have different values since the imagePath is included.
Though looking at the sed commands, I'm thinking some of those aren't correct even for Enterprise. Like `sed -ie "s?quay.io.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" manifests/04-deployment-prometheus-operator.yaml` doesn't look correct to me.

I'd be fine with switching to a Note for the ImagePath and reducing to 1 doc but it would be expecting the user to make a lot of updates themselves to include/replace the imagePaths.